packages/terminal/src/browser/terminal-widget-impl.ts

```diff
@@ -101,16 +95,30 @@ export class TerminalWidgetImpl extends TerminalWidget implements StatefulWidget
 
         this.term = new Xterm.Terminal({
             experimentalCharAtlas: 'dynamic',
+            experimentalBufferLineImpl: 'TypedArray',
```

Comment:
this is default with the currently used version, and the setting is deprecated/gone at least on master of xtermjs

Reply:
ok, will remove again